### DISCUSSION TITLE
Use and bundle Qdrant in a desktop app

### DISCUSSION THREAD

**Discussion by martinnormark** [qdrant/qdrant#1446] [General]
Hi,

Many use cases with Qdrant are obviously cloud/server based.

But what if I wanted to create a desktop app for e.g. lawyers to help them reason about large amounts of documents, but for privacy concerns do not want that moved to a server.

Two questions:

1. I presume bundling Qdrant in a desktop app is not prohibited by the license, is that correct?
2. Any examples on building as a runnable binary cross platform? I presume the [from source](https://qdrant.tech/documentation/install/#from-source) path is the way to go, but is it prepared to run on Windows as well?

**Comment by generall** [comment]
> I presume bundling Qdrant in a desktop app is not prohibited by the license, is that correct?

Yes, you can modify and use it without restrictions

> Any examples on building as a runnable binary cross platform? I presume the [from source](https://qdrant.tech/documentation/install/#from-source) path is the way to go, but is it prepared to run on Windows as well?

Since v1.0 it works on windows, you can use cargo to build it in a pretty same way as you do on linux. There are a few dependencies you might need to install, like protoc, llvm, e.t.c.
